subsys/dfu/dfu_target/Kconfig

```diff
@@ -121,10 +121,54 @@ config DFU_TARGET_SUIT
 	select SUIT_UTILS
 	select ZCBOR
 	select ZCBOR_CANONICAL
+	select FLASH
+	select STREAM_FLASH
+	select STREAM_FLASH_ERASE
 	select DFU_TARGET_STREAM
+	# Minimal SUIT configuration to support SUIT processing
```

Comment:
You don't need most of thesethese configuration options.
I should have https://github.com/nrfconnect/sdk-nrf/pull/17428 where I add a new configuration, `SUIT_DFU_CANDIDATE_PROCESSING_PUSH_TO_CACHE` - you only need to copy the options from there

Reply:
I needed this config so much! Because FULL_PROCESSING is too large, and MINIMAL_PROCESSING is not enough. :)

Reply:
I will wait for it.

Reply:
But wait, it is only for CACHE_PROCESSING, and what about the case if we don't want to cache processing, but still we need to process the envelope? I still need those configurations, but without:
`select SUIT_CACHE`
`select SUIT_CACHE_RW`

Reply:
I discussed it with @SylwesterKonczyk and I will propose a better solution.

Reply:
Ok, reworked, now it requires SUIT_DFU_CANDIDATE_PROCESSING_FULL or SUIT_DFU_CANDIDATE_PROCESSING_PUSH_TO_CACHE.